### PR TITLE
Closes #566 Memory leak in SohoFileUploadComponent

### LINKED ISSUE #566: Memory leak in SohoFileUploadComponent
**Describe the bug**
Unreleased event listener in jQueryElement

**To Reproduce**
1. Use SohoFileUploadComponent in a component
2. Navigate from the component to trigger destroy in SohoFileUploadComponent 
3. Create Heap snapshot from Chrome memory tab
4. filter results by 'fileupload'

**Expected behavior**
No leaking FileUpload objects

**Version**
- ids-enterprise-ng: [e.g. v4.9.0 or v4.10.0]

**Screenshots**
Heap snapshot:
image

**Platform**
 - OS Version: Windows10
 - Browser Name: Chrome
 - Browser Version: 76.0.3809.87 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by nbcp**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Memory leak in SohoFileUploadComponent

**Related github/jira issue (required)**:
Closes #566 

**Steps necessary to review your pull request (required)**:
image

### CODE REVIEW

**Comment by tmcconechy** [comment]
@pwpatton do we want this on the 5.5.x branch?

File: projects/ids-enterprise-ng/src/lib/fileupload/soho-fileupload.component.ts [hunk 1]
```diff
@@ -119,6 +119,8 @@ export class SohoFileUploadComponent implements AfterViewInit, OnDestroy {
       this.fileUpload.destroy();
       this.fileUpload = null;
     }
+
+    this.jQueryElement.off();
```
Comment (pwpatton) [inline projects/ids-enterprise-ng/src/lib/fileupload/soho-fileupload.component.ts]: probably want to add safety check

```
if (this.jQueryElement) {
    this.jQueryElement.off();
}
```

Comment (tmcconechy) [inline projects/ids-enterprise-ng/src/lib/fileupload/soho-fileupload.component.ts]:
```suggestion
if (this.jQueryElement) {
    this.jQueryElement.off();
}
```

**Comment by pwpatton** [comment]
Yes I think we do want this in 5.5

**Comment by tmcconechy** [comment]
Ok will have to recreate the the PR as changing the base branch wont work.

**Comment by tmcconechy** [comment]
Closing as @pwpatton will redo the PR to 5.5.x

**Pull request closed without merging**